src/platform/landing-pages/dev-template.ejs

```diff
@@ -5,9 +5,10 @@
     <meta name="HandheldFriendly" content="True">
     <meta name="MobileOptimized" content="320">
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><%= htmlWebpackPlugin.options.title %></title>
 
     <script>
-      {% include "src/site/assets/js/record-event.js" %}
+      <%= inlineScripts['record-event.js'] %>
```

Comment:
The previous format was Liquid template code and threw errors. This is just converting it to the templating language expected by HtmlWebpackPlugin (which is based on Lodash templates by default).

Lodash templates are fairly limited, and there is no analog for `include`, so I made do with an object that references the relevant scripts.